blocks/measuring-impact-cards/measuring-impact-cards.js

```diff
@@ -0,0 +1,66 @@
+import { div, a, p } from '../../scripts/dom-helpers.js';
+import { moveInstrumentation, fetchLanguagePlaceholders } from '../../scripts/scripts.js';
+
+const getShortendIndicatorName = (indicatorTag, placeholdersData) => {
+  // Return Same indicator tag content if hyphen is not present
+  if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;
```

Comment:
move generic code in a function to return array
` if (indicatorTag && indicatorTag.indexOf('-') < 0) return indicatorTag;

  const indicatorKey = indicatorTag.split('-')[0].trim().replace(/[^a-zA-Z0-9]/g, '');`

Reply:
Done